tests/Fixtures/PostFormation.php

```diff
@@ -139,6 +142,18 @@ public function slices(): array
         ];
     }
 
+    public function actions(): array
+    {
+        return [
+            Action::make('set-status')
+                ->job(SetStatus::class)
+                ->can('setStatus', Post::class)
+                ->fields(['status' => 'active'])
```

Comment:
The fields() method allows the developer to define validation rules for fields posted to the endpoint

So lets tweak this to 

```suggestion
                ->fields(['status' => ['in:active,draft'])
```
---

And lets add a test that posts the following and [assertInvalid](https://laravel.com/docs/8.x/http-tests#assert-invalid)

```php
$this->post('the-endpoint', [
    'selected' => 'all',
    'fields' => [
         'status' => 'invalid-status-type'
    ]
])
```

And in the existing tests we can add the status `active`

Reply:
Also, lets make the fields array in the job 
```php
request()->only(array_keys($action->fields))
```

Reply:
Done